src/components/ApiCaller/ApiCaller.js

```diff
@@ -93,7 +93,9 @@ class ApiCaller {
         }
       };
 
-      request.onerror = () => {
+      request.onerror = (result) => {
+        NotificationActions.notify("Request failed, there is a problem with the connection.", "error")
```

Comment:
Please change message to "Request failed, there might be a problem with the connection to the server."